### PR TITLE
NO-JIRA: Added P4A and Floe logos to footer and contrast styling.

### PR DESCRIPTION
**Pull request by jhung**
@dayotte or @jobara can you verify that it looks okay?

### CODE REVIEW

File: src/documents/css/idg.css.styl [hunk 1]
```diff
@@ -499,10 +514,21 @@ contrast-theme(foregroundColor, backgroundColor) {
...
+    footer svg {
+        stroke: foregroundColor;
+        fill: backgroundColor;
+    }
+
+    /* Special styling for P4All logo */
+    svg #prosperity, svg #tree {
+        stroke: backgroundColor;
+        fill: foregroundColor;
+    }
```
Comment (jobara) [inline src/documents/css/idg.css.styl]: This looks similar to styling from previous in the file. Can this duplication be factored out?

**Comment by jobara** [comment]
@dayotte Here are some screenshots of how the footer looks. I've include one with and without the mobile layout.

@jhung I think that logos seem a bit big, can we make them smaller. Maybe push them off to the sides in the desktop layout and inline with the project links in the mobile?

Desktop:
desktop

Mobile:
mobile

**Comment by dayotte** [comment]
Thanks @jobara. I agree, let's make them smaller - maybe 50% @jhung?

**Comment by jobara** [comment]
@jhung i did a quick look over the CSS changes and they seemed okay.

**Comment by dayotte** [comment]
@jhung I like the size of the logos now, thanks. @jobara and I also talked about saving vertical space by removing the line "Supported by ..." at the bottom -  and then we could add links and alt-text to the logos instead. Also, we thought it would be good (at desktop screen size) to remove the line break between "Created by the community members of the" and "Inclusive Design Research Centre ...". I can make these changes if that sounds good to you.

**Comment by jhung** [comment]
If you have time to do that, that would be great. You can make those changes to your local branch and push it up. I'll then merge it and take a look.

**Comment by jhung** [comment]
Replaced by PR14: https://github.com/inclusive-design/guide.inclusivedesign.ca/pull/14

**Pull request closed without merging**